### ISSUE TITLE
No alt text/aria-label

### ISSUE THREAD

**Issue by fosterc1** [cgkineo/adapt-hotgrid#82]
The alt text that has been added to the hotgrid Item Graphic is not being displayed
![2022-06-03_09-24-51](https://user-images.githubusercontent.com/29701150/171817727-4dc59326-00b4-437e-94ae-d08aa2248dbb.jpg)
![2022-06-03_09-23-46](https://user-images.githubusercontent.com/29701150/171817737-23f8e79a-fa97-40b7-94d2-dffeeb56e2ab.jpg)
.

Many thanks

**Comment by fosterc1** [comment]
Done, thanks
